lightning/src/ln/msgs.rs

```diff
@@ -2052,7 +2059,7 @@ mod tests {
 		do_encoding_channel_update(true, true, true, true);
 	}
 
-	fn do_encoding_open_channel(random_bit: bool, shutdown: bool) {
+	fn do_encoding_open_channel(random_bit: bool, shutdown: bool, chan_type: bool) {
```

Comment:
channel_type is a bitmap or feature set. The boolean should be named `encode_channel_type` or `should_encode_channel_type` or sth. along those lines.

Reply:
went with "incl" ie "include".